### PR TITLE
Task #169: update GitHub templates

### PR DESCRIPTION
**Pull request by martijncalker**
Update the existing GitHub templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -2,16 +2,32 @@
...
 you've considered.
 
-**Additional context** Add any other context or screenshots about the feature request here.
+## Breaking stuff and backwards compatibility
+
+Will the new feature break current environments and support backwards compatibility? 
+
+## Additional context
+
+If applicable, add any other context about the problem here.
```
Comment (j-zimnowoda) [inline .github/ISSUE_TEMPLATE/feature_request.md:33]: Aren't first four items part of commit message?

feat: add new feature
BREAKING CHANGE: this is breaking change

OR

fix: change sth
BREAKING CHANGE: this is breaking change

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -2,16 +2,32 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: ''
+labels: 'feature'
 assignees: ''
 ---
 
-**Is your feature request related to a problem? Please describe.** A clear and concise description of what the problem
```
[the remark this answers is not in the document]
Comment (j-zimnowoda) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: Not needed. 
The problem section is described below:
> Briefly describe the feature you want to see added to Otomi, preferably in the form of a user-story (e.g. "As a [persona],

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Nice, I have only one remark. Please decide what is the best.

**Comment by Morriz** [comment]
closing as this is mostly done via direct template edits

**Pull request closed without merging**